src/queries.jl

```diff
@@ -75,14 +75,14 @@ julia> queryall(r, :symbol, ❓, >(5))
 """
 queryall(args...; filo=true, kwargs...) = query(args..., Val{true}(); filo, kwargs...)
 
-
 """
 $TYPEDSIGNATURES
 
 A query function searching for the first slot in a register that has a given tag.
 
 Wildcards are supported (instances of `Wildcard` also available as the constants [`W`](@ref) or the emoji [`❓`](@ref) which can be entered as `\\:question:` in the REPL).
 Predicate functions are also supported (they have to be `Int`↦`Bool` functions).
+The order of query lookup can be specified in terms of FIFO or FILO and defaults to FIFO if not specified.
```

Comment:
I think the default is FILO? Not FIFO? At least that is what the kwargs seem to say.